sphinx/developers/commit-testing.rst

```diff
@@ -5,17 +5,34 @@ Automated tests
 ---------------
 
 The :source:`Bio-Formats testing framework <components/test-suite>` component
-contains most of the infrastructure to run automated tests against the data
-repository.
-
-After checking out source code and building all the JAR files (see
-:doc:`building-bioformats`), switch to the :file:`test-suite` component and run the tests using the :program:`ant` ``test-automated`` target::
+contains the infrastructure to run automated tests against reference data.
+These tests check the metadata and binary data of every dataset using
+INI-based configuration files as the source of truth.
+
+All representative files submitted as part of the `bug report <bug-reporting>`_
+or development projects and used as supporting data for new Bio-Formats releases
+are stored in a curated QA repository, also known as the full data repository,
+managed by the Bio-Formats maintainers alongside the associated configuration
+files. As part of the review process, every change against Bio-Formats is
+tested nightly against the entire data repository and all tests must be
+passing in order for a contribution to get integrated.
```

Comment:
Is it worth stating explicitly here that the full data repository and associated configuration cannot be shared? And maybe that a subset is available from https://downloads.openmicroscopy.org/images/?

Reply:
Thanks I debated where to define this explicitly and I was erring on the side of adding this information to https://ome-contributing.readthedocs.io/en/latest/third-party-policy.html and possibly linking from here.

Reply:
Linking to https://ome-contributing.readthedocs.io/en/latest/third-party-policy.html once it's been updated sounds fine.